packages/eslint-plugin/src/rules/no-keyframes-tagged-template-expression/index.ts

```diff
@@ -5,6 +5,8 @@ import { createNoTaggedTemplateExpressionRule, isKeyframes } from '../../utils';
 export const noKeyframesTaggedTemplateExpressionRule: Rule.RuleModule = {
   meta: {
     docs: {
+      recommended: true,
```

Comment:
Adding a few `recommended: true` _could_ be a problem, but this is a minor bump on a `0.x` version which means major to me, LGTM if Eurybia is happy!